ansible/roles/base/tasks/main.yml

```diff
@@ -17,18 +15,23 @@
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
-      apt: name=python-pip
+      apt: 
+       name: python3-pip
       become: yes
+      become_method: sudo
 
-    - name: install python-virtualenv
-      apt: name=python-virtualenv
+    - name: Install python-virtualenv
+      apt:
+       name: python-virtualenv
       become: yes
 
     - name: Install PIP requirements
+      become: yes
```

Comment:
Why are we adding these? You shouldn’t need them as part of the pip install tasks. When you try and sudo install them, they get installed as a super user… if you leave off the become / become -H it should just work. Were you having issues?